web-registry/src/pages/Projects/hooks/useProjectsSellOrders.tsx

```diff
@@ -19,6 +20,7 @@ type Props = {
   sellOrders?: SellOrderInfoExtented[];
   regenPrice?: number;
   limit?: number;
+  userAddress?: string;
```

Comment:
this `userAddress` that is defined here, as an argument in `useProjectsSellOrders` , is not actually used in the hook function... since the `wallet.address` is being accessed and passed to the `getProjectDisplayData` function

Reply:
Forgot to remove it, thanks!